dracut.conf.d/50-no-xattr.conf.example

```diff
@@ -0,0 +1,2 @@
+# disable xattr
+DRACUT_NO_XATTR=1
```

Comment:
Not to mention the convenience of this new approach of educating users to add environment variables as configuration options... I wonder how an [external C binary](https://github.com/dracut-ng/dracut-ng/blob/1e802f15fee3d6402d83e0efefef8bb88c5a33e3/src/install/dracut-install.c#L2299) can get this value without an `export`... because this variable is not being exported anywhere in `dracut.sh`. Does anyone tested that this _really_ works?